library/cohesity_restore_vm.py

```diff
@@ -703,7 +703,7 @@ def main():
                     token=job_details['token'],
                     type="kRecoverVMs",
                     vmwareParameters=dict(
-                        poweredOn=module.params.get('power_state'),
+                        poweredOn=module.params.get('power_state')=="yes",
```

Comment:
If this is checking boolean, other params should too ?

Reply:
Removed.